gridstatus/spp.py

```diff
@@ -1216,6 +1216,102 @@ def _match(
             )
         ]
 
+    ### NEW
+    @support_date_range("DAY_START")
+    def get_hourly_load(self, date, end=None, verbose=False):
+        """Get Hourly Load
+
+        Supports recent data. For historical annual data use get_hourly_load_annual
```

Comment:
It looks like this method doesn't support "today" or "latest" since the data is released on a delay of 2 days. We should raise a `NotSupport` exception in these cases.

Reply:
Updated: [172f553](https://github.com/kmax12/gridstatus/pull/406/commits/172f5538cf5508b655392af2cde08cb95ebc4b1f)